Gordon360/Services/NewsService.cs

```diff
@@ -302,7 +302,7 @@ private string GetImageURL(string filename)
         {
             var serverAddress = _serverUtils.GetAddress();
             if (serverAddress is not string) throw new Exception("Could not upload Student News Image: Server Address is null");
-            var url = $"{serverAddress}/browseable/uploads/news/{filename}";
+            var url = $"{serverAddress}browseable/uploads/news/{filename}";
```

Comment:
@EjPlatzer did you test this on localhost to make sure it works? I have a feeling that `_serverUtils.GetAddress()` is returning `http://localhost` on local dev, but `https://360api.gordon.edu/` in prod. Note the slash at the end.

Reply:
No, I did not test it locally but I agree with your assessment. This was a hotfix to make sure it's fixed in production ASAP. We can rethink `_serverUtils.GetAddress` later to make sure it still works locally.